Snapper.Core/UpdateTestSnapshot.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 
-namespace Snapper.Json.Xunit
+namespace Snapper.Core

```

Comment:
Hmm I'm not sure that this attribute belongs in the core project. Its only being used in the XUnit and NUnit packages.

Maybe leave this in the XUnit project and create a new one in the NUnit project?

Reply:
Sounds fine to me!